python/hail/table.py

```diff
@@ -476,14 +476,42 @@ def _select_globals(self, caller, s):
                       key=table_key_type,
                       n_partitions=nullable(int))
     def parallelize(cls, rows, schema=None, key=None, n_partitions=None):
-        rows = to_expr(rows, hl.tarray(schema) if schema is not None else None)
+        """Parallelize a local array of structs into a distributed table.
+
+        Examples
+        --------
+        Parallelize a list of dictionaries:
+
+        >>> a = [ {'a': 5, 'b': 10}, {'a': 0, 'b': 200} ]
+        >>> table = hl.Table.parallelize(hl.literal(a, 'array<struct{a: int, b: int}>'))
+        >>> table.show()
+
+        Warning
+        -------
+        Parallelizing very large local arrays will be slow.
+
+        Parameters
+        ----------
+        rows
+            List of row values, or expression of type ``array<struct{...}>``.
+        schema : str or :class:`.HailType:`, optional
+            Value type.
+        key : Union[str, List[str]]], optional
+            Key field(s).
+        n_partitions : int, optional
+
+        Returns
+        -------
+        :class:`.Table`
+        """
+        rows = hl.literal(rows, dtype=hl.tarray(schema) if schema is not None else None)
```

Comment:
will this work if `rows` is originally an expression? It looks like it would just wrap the entire expression in an `ir.Literal` node? If this doesn't work currently, you may want to take that line out of the documentation for now.

Reply:
`literal` evaluates any expressions passed into it, so this will work

Reply:
caught a few bugs in this too

Reply:
oh, nice.

Reply:
actually, this should be `to_expr`, not literal.

Reply:
yeah, that's totally the right thing. to_expr doesn't force eager evaluation if there's an expr somewhere inside, whereas literal does.